tensorboard/components/tf_scalar_dashboard/tf-scalar-chart.html

```diff
@@ -0,0 +1,316 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+<link rel="import" href="../paper-icon-button/paper-icon-button.html">
+
+<!-- Prototype framework code. This will end up in its own file. -->
+<!--
+  A compact heading to appear above a single visualization, often
+  corresponding to either a single tag or a single run-tag combination.
+
+  Properties:
+    `title` does just what it says on the tin.
+    `color` can be set to display a colored border at the left of the
+        card; if left unset, no border will be displayed:
+    Any contents of the heading will be rendered below the title, and
+        can be used to display a subtitle, some small control widgets,
+        or similar.
+-->
+<dom-module id="tf-card-heading">
```

Comment:
Maybe split this into a separate file? Easier for when you refactor it back into a shared component, too.